spring-integration-redis/src/test/java/org/springframework/integration/redis/RedisTest.java

```diff
@@ -33,36 +38,64 @@
 import org.springframework.integration.test.util.TestUtils;
 import org.springframework.messaging.Message;
 
+import io.lettuce.core.ClientOptions;
+import io.lettuce.core.SocketOptions;
+
 /**
+ * The base contract for all tests requiring a Redis connection.
+ * The Testcontainers 'reuse' option must be disabled,so, Ryuk container is started
+ * and will clean all the containers up from this test suite after JVM exit.
+ * Since the Redis container instance is shared via static property, it is going to be
+ * started only once per JVM, therefore the target Docker container is reused automatically.
+ *
+ * @author Artem Vozhdayenko
  * @author Oleg Zhurakousky
  * @author Gary Russell
  * @author Artem Bilan
  *
+ * @since 6.0
  */
-public abstract class RedisAvailableTests {
+@Testcontainers(disabledWithoutDocker = true)
+public interface RedisTest {
```

Comment:
You know I call them like this: `MySqlContainerTest`, `MosquittoContainerTest`, `LocalstackContainerTest` etc...

What was the reason to avoid a `Container` indication for this interface?

Reply:
No specific reason except my inattentiveness 😄